containers/deploy/pulp/deployments.yml

```diff
@@ -129,6 +145,8 @@
                       containers:
                         - name: pulp-worker
                           securityContext: {}
+                          serviceAccount: anyuid
+                          serviceAccountName: anyuid
```

Comment:
I don't think this goes inside the container spec. If we don't need it we can remove it from there and below.